windows/autorelease_windows.ps1

```diff
@@ -7,8 +7,11 @@ echo "Install Qt..."
 # Install Qt
 pip install aqtinstall
 echo "Get Qt 64 bit..."
+# intermediate solution if the main server is down: append e.g. " -b https://mirrors.ocf.berkeley.edu/qt/" to the "aqt"-line below
```

Comment:
A fallback argument has been added to aqt recently. Maybe we could use this?

Reply:
good idea. I hop I will not forget this.
Likely I will make the qt-install in Android build aqt based, too